_includes/_footer.htm

```diff
@@ -1,9 +1,9 @@
-<footer class="gridcoin-footer h-100" data-background-color="black">

```

Comment:
why was the h-100 class removed here? Isn't it a valid bootstrap 4 class? See https://getbootstrap.com/docs/4.0/utilities/sizing/

Reply:
The h-100 was part of the previous low content fix, It's not required anymore.